torchgeo/datasets/western_usa_live_fuel_moisture.py

```diff
@@ -0,0 +1,353 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Western USA Live Fuel Moisture Dataset."""
+
+import glob
+import json
+import os
+from typing import Any, Callable, Optional
+
+import torch
+from torch import Tensor
+
+from .geo import NonGeoDataset
+from .utils import download_radiant_mlhub_collection, extract_archive
+
+
+class WesternUSALiveFuelMoisture(NonGeoDataset):
```

Comment:
Could also make it a GeoDataset if you want, that's what I did for GBIF/iNaturalist/EDDMapS. Depends on how you intend to use the dataset and whether or not you need to combine it with other data sources.

Reply:
At the moment I think I want to use just by itself and index the datapoints in "standard tabular machine learning" fashion. I also discovered that there is an image-based version from the corresponding paper: https://github.com/kkraoj/lfmc_from_sar so looking to add that as well maybe